tests/dags/providers/provider_api_scripts/test_museum_victoria.py

```diff
@@ -132,7 +131,6 @@ def test_get_image_data_large():
     assert actual_height == 2581
     assert actual_width == 2785
     assert actual_filesize == 890933
-    assert actual_filetype == "jpg"
```

Comment:
Were these checks removed because the image store is handling this now?

Reply:
Yes, but I think I should re-write the tests to use `save_item` to see if the filetype is actually saved, just like I did in other PRs.